windows builds failing

### Steps to Reproduce

tried cloning from both main and 0.33.0
clean build and install, both debug and release build, same issue on cargo install

   Compiling rye v0.33.0 (C:\Users\XXX\dev\rye\rye)
error[E0432]: unresolved import `winapi::um::ioapiset`
   --> rye\src\cli\shim.rs:113:21
    |
113 |     use winapi::um::ioapiset::DeviceIoControl;
    |                     ^^^^^^^^ could not find `ioapiset` in `um`

error[E0432]: unresolved import `winapi::um::winioctl`
   --> rye\src\cli\shim.rs:115:21
    |
115 |     use winapi::um::winioctl::FSCTL_GET_REPARSE_POINT;
    |                     ^^^^^^^^ could not find `winioctl` in `um`


### Expected Result

That build succeeds

### Actual Result

builds fails as above

If I update the rye/Cargo.toml with the 2 required features then it builds and installs

[target."cfg(windows)".dependencies]
winapi = { version = "0.3.9", default-features = false, features = ["winuser","winioctl","ioapiset"] }
winreg = "0.52.0"


### Version Info

rye 0.33.0 and main

### Stacktrace

as above

--- a/rye/Cargo.toml
+++ b/rye/Cargo.toml
@@ -60,7 +60,7 @@ dotenvy = "0.15.7"
 xattr = "1.3.1"
 
 [target."cfg(windows)".dependencies]
-winapi = { version = "0.3.9", default-features = false, features = ["winuser"] }
+winapi = { version = "0.3.9", default-features = false, features = ["winuser", "winioctl", "ioapiset"] }
 winreg = "0.52.0"
 
 [target."cfg(windows)".build-dependencies]

